grammar.js

```diff
@@ -551,8 +558,21 @@ module.exports = grammar({
     nchar: $ => parametric_type($, $.keyword_nchar),
     nvarchar: $ => parametric_type($, $.keyword_nvarchar),
 
+    _time_zone: $ => seq(
```

Comment:
```suggestion
    _include_time_zone: $ => seq(
```
this is clearer imo, but I definitely like the more detailed deconstruction of timestamps overall!

Reply:
Fine by me.